src/lib/bootloader/stage1_device.rb

```diff
@@ -102,18 +106,18 @@ def underlaying_devices_one_level(dev)
     #
     # Ignores physical volumes on a whole disk. See bnc#980529
     #
-    # @param volumes_group_list [Y2Storage::LvmVgsList]
+    # @param volume_groups_list [Y2Storage::LvmVgsList]
     # @return [Y2Storage::LvmPvsList]
     def usable_pvs(volume_groups_list)
-      pvs = volume_groups_list.physical_volumes
-      pvs.with { |pv| !Storage.disk?(pv.blk_device) }
+      pvs = volume_groups_list.reduce([]) { |a, e| a.concat(e.lvm_pvs) }
+      pvs.reject { |pv| pv.blk_device.is?(:disk) }
```

Comment:
As already mentioned, there is not need anymore to receive a list here. That was a trick. In fact this method makes sense for one concrete VG.
```ruby
def usable_pvs(volume_group)
  volume_group.lvm_pvs.reject { |pv| pv.plain_blk_device.is?(:disk) }
end